src/clr-angular/forms/common/layout.ts

```diff
@@ -12,6 +12,7 @@ import { Layouts, LayoutService } from './providers/layout.service';
 })
 export class ClrLayout implements OnInit {
   @Input('clrLayout') layout: Layouts;
+  @Input('clrLabelSize') labelSize: number;
```

Comment:
We can just set a default value here and then the logic below can always assume a value. Doing the following is probably better so we can better handle dynamic inputs.

```
private _labelSize: number = 2;
@Input('clrLabelSize') 
set labelSize(size: number) {
  if (this.layoutService.labelSizeIsValid(size) {
    this._labelSize = size;
  }
}
get labelSize(): number {
  return this._labelSize;
}
```

Reply:
This was my initial approach but I decided to keep it consistent with the layout logic. Thanks!

Reply:
Ok I missed this the first time, but this stuff needs to go onto the ClrForm and not ClrLayout. 

Also let's put the value directly on the service regardless of what layout, because it only gets used for horizontal/vertical anyways so storing it is fine.

Reply:
horizontal/vertical? The form's css grid options only allow horizontal/compact